### PR TITLE
Script to remove expired sessions.

### PR DESCRIPTION
**Pull request by dracos**
This also can set up users so that the admin
"Log user out" function works correctly.

### CODE REVIEW

File: bin/expire-sessions [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+use Storable;
+
+GetOptions(
+    'init' => \my $init,
+);
+
+my $rs = FixMyStreet::DB->resultset("Session");
+
+# Delete expired sessions (including from in User object)
+while (my $session = $rs->search({ expires => { '<', time() - 86400*7 } })->next) {
```
Comment (struan) [inline bin/expire-sessions]: might be nice to include a comment here saying why we are using expired a week ago as the criteria.

I assume just to give a bit of leeway?

File: bin/expire-sessions [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    while (my $session = $rs->next) {
+        my $user = get_user($session) or next;
+        my $id = get_id($session);
+        my $sessions = $user->get_extra_metadata('sessions');
+        my @new_sessions = uniq @$sessions, $id;
+        update_user_sessions($user, \@new_sessions) if @new_sessions != @$sessions;
+    }
+}
+
+# ---
```
Comment (struan) [inline bin/expire-sessions]: ?

File: bin/expire-sessions (hunk 1 above)
Comment (dracos) [reply bin/expire-sessions]: Yeah, dunno why TBH (db should ignore any past expiry), will change to just be now.

**Comment by dracos** [comment]
Merged as 5260280b60a1192f87987c34a78fce63fc5c1d26 (added line to crontab).

**Pull request closed without merging**